gui-apps/foot-terminfo/foot-terminfo-1.10.3.ebuild

```
@@ -0,0 +1,37 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="https://codeberg.org/dnkl/foot/archive/${PV}.tar.gz  -> foot-${PV}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/foot"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://codeberg.org/dnkl/foot.git"
+fi
+
+DESCRIPTION="Terminfo for foot, a fast, lightweight and minimal Wayland terminal emulator"
+HOMEPAGE="https://codeberg.org/dnkl/foot"
+LICENSE="MIT"
+SLOT="0"
+
+DEPEND=""
+RDEPEND="<sys-libs/ncurses-6.3"
```

Believe this is meant to be a blocker.
```suggestion
RDEPEND="!>=sys-libs/ncurses-6.3"
```